src/momento-redis-client.ts

```diff
@@ -131,10 +132,15 @@ export class MomentoRedisClient
     return this._isOpen;
   }
 
-  public async connect(): Promise<void> {
-    // TODO this should be ping once it is in the client
-    await this.get('sRITPPymF1yEB6rFizrI0ZeCMq012uXFjBNRNokAv4');
+  public async connect(): Promise<
+    RedisClientType<RedisModules, RedisFunctions, RedisScripts>
+  > {
+    await this.client.ping();
     this._isOpen = true;
+    return new MomentoRedisClient(
+      this.client,
+      this.cacheName
+    ) as unknown as RedisClientType<RedisModules, RedisFunctions, RedisScripts>;
```

Comment:
@cprice404 wdut think here? Seems like they updated package to have `connect` return instance of client.

Wondering if should be this instead?
```typescript
  public async connect(): Promise<
    RedisClientType<RedisModules, RedisFunctions, RedisScripts>
  > {
    await this.client.ping();
    this._isOpen = true;
    return this as unknown as RedisClientType<
      RedisModules,
      RedisFunctions,
      RedisScripts
    >;
  }
```

Reply:
yes I agree you should just return `this` rather than creating a new one. nice catch on noticing that they'd changed their function signature.